apps/frontend/src/pages/analytics/analytics.tsx

```diff
@@ -68,6 +68,10 @@ const Analytics = (): JSX.Element => {
 		}
 	}, [dateRangeValue, isDirty, handleFormSubmit]);
 
+	const visibleActivityLogs = activityLogs.filter(
+		(log) => !log.contributorIsHidden,
+	);
```

Comment:
this should be filtered inside backend query

Reply:
Is it mandatory to modify query like this `builder.select("id", "name").where("isHidden", false);` or can it be just check in service 			`if (!contributor.isHidden)`, not sure it is needed to add isHidden to contributor on `GitEmailModel`.

(I'll push it with just a check for a better look)

Reply:
we don't need isHidden in GitEmailModel for now, it's better to use query builder